File: Problem Set 1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int getHeight(void);
void printPyramid(int n);

int main(void)
{
    int n = getHeight();
    printPyramid(n);
}

int getHeight(void)
{
    int n;
    do
    {
        n = get_int("Height:");
    }
    while (n < 1);

    return n;
}

void printPyramid(int n)
{
    for (int row = 1; row <= n; row++)
    {
        for (int j = 0; j < n - row; j++)
        {
            printf(" ");
        }
        for (int k = 0; k < row; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Problem Set 6/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if 0 < height < 9:
        break

for i in range(height):
    spaces = height - 1
    while spaces > i:
        print(" ", end="")
        spaces -= 1
    for j in range(i + 1):
        print("#", end="")
    print("  ", end="")

    for k in range(i + 1):
        print("#", end="")
    print()
